corehq/apps/reports/util.py

```diff
@@ -190,6 +190,11 @@ class SimplifiedUserInfo(
     def group_ids(self):
         return Group.by_user(self.user_id, False)
 
+    @property
+    @memoized
+    def location_id(self):
```

Comment:
It's used in both the user rows and the total row, so I kept it to cut down on those calls, but if it still seems like overkill I can change that.